pkgs/top-level/python-packages.nix

```diff
@@ -126,6 +126,8 @@ let
     mpi = pkgs.openmpi;
   };
 
+  bootstrapped-pip = callPackage ../development/python-modules/bootstrapped-pip { };
```

Comment:
possible reason could be to not confuse newbies which pip to use.
another would be that our all-packages.nix is growing and if we can take something from it we should.